includes/Experiments.php

```diff
@@ -394,17 +394,6 @@ public function get_experiments(): array {
 				'group'       => 'editor',
 				'default'     => true,
 			],
-			/**
-			 * Author: @miina
-			 * Issue: #10113
-			 * Creation date: 2022-02-22
-			 */
-			[
```

Comment:
We usually first enable a feature by default before removing the flag code completely. But I suppose for this small change here it's fine.

Reply:
I know, I wasn't planning to add it behind a feature flag originally at all since it's quite a small change but since UX preferred having it released together with this feature (Applying presets), then added it anyway for ensuring these two get released together.